actions/delete_global_data_MOD.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-undef */
```

Comment:
you can put these on the same line 
`/* eslint-disable no-undef , no-unused-vars*/`